src/pages/lessons/[slug].tsx

```diff
@@ -383,10 +385,6 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
     setLessonMaxWidth(Math.round((height - HEIGHT_OFFSET) * 1.77))
   }, [height])
 
-  React.useEffect(() => {
```

Comment:
I didn't understand this and don't think it is needed as `useEggheadPlayer` prefs manage this state so no need to duplicate it.